README.md

```diff
@@ -32,6 +32,59 @@ In addition to the in-library decoding support, you may use the included `plcras
 `plcrashutil convert --format=iphone example_report.plcrash | symbolicatecrash`
 Future library releases may include built-in re-usable formatters, for outputting alternative formats directly from the phone.
 
+## Usage
+
+PLCrashReporter can be added to your app via [CocoaPods](https://guides.cocoapods.org/using/using-cocoapods.html), [Carthage](https://github.com/Carthage/Carthage#quick-start), [Swift Package Manager](https://developer.apple.com/documentation/xcode/adding_package_dependencies_to_your_app), or by manually adding the binaries to your project.
+
+### Example
+
+The following example shows a way how to initialize crash reporter. Please note that enabling in-process crash reporting will conflict with any attached debuggers.
+
+```objc
+@import CrashReporter;
+
+...
+
+// It is strongly recommended that local symbolication only be enabled for non-release builds.
+// Use PLCrashReporterSymbolicationStrategyNone for release versions.
+PLCrashReporterConfig *config = [[PLCrashReporterConfig alloc] initWithSignalHandlerType: PLCrashReporterSignalHandlerTypeMach
```

Comment:
Does it make sense to clarify the flags `PLCrashReporterSignalHandlerTypeMach` and `PLCrashReporterSymbolicationStrategyAll` and possible values?

Also, can we put the selection of `PLCrashReporterSymbolicationStrategyAll` and `PLCrashReporterSymbolicationStrategyNone` into the condition `#if DEBUG` to show the comment idea?

Reply:
It is documented in the enum. No need for it in README.

No `DEBUG` is not requirement here, it's just recommendation.